akka-components/src/main/scala/ClusterComponent.scala

```diff
@@ -415,11 +415,17 @@ object ClusterComponent {
     trait JsonEntityId {
       _: ShardedT =>
 
+      /**
+       * The placeholder that will replace any | in the produced String as this character is illegal due to Akka's entityId encoding.
+       * When overriding keep the new placeholder permanently or existing entities won't be resolved.
+       * */
+      val barPlaceholder = "❘❘bar❘❘"
```

Comment:
These are UTF-8 bars

Reply:
interesting approach. this comment might be useful in the code comment too (as one might wonder why a single | is an issue but a || isn't (as they look very similar)

Reply:
added